File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PackagesUpdateUseByDateRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PackagesUpdateUseByDateRequest {
    @JsonProperty("ExpirationDate")
    public String expirationDate;
    @JsonProperty("Label")
    public String label;
    @JsonProperty("SellByDate")
    public String sellByDate;
    @JsonProperty("UseByDate")
    public String useByDate;
}

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/PackagesUpdateUseByDateRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class PackagesUpdateUseByDateRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("ExpirationDate")]
        public string? ExpirationDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Label")]
        public string? Label { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("SellByDate")]
        public string? SellByDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("UseByDate")]
        public string? UseByDate { get; set; }
    }
}
